src/info/info.ts

```diff
@@ -494,20 +494,16 @@ port.onMessage.addListener(async (message) => {
     }
     Array.from(document.getElementsByTagName("audio")).map(i => new Plyr(i));
 
-    const footer = document.getElementById("footer") as HTMLElement;
-    if (footer !== undefined && serverUrl !== undefined && request !== undefined) {
-        const feedbackAnchor = document.createElement("a");
-        feedbackAnchor.textContent = browser.i18n.getMessage("feedbackLinkText");
+    const feedbackAnchor = document.getElementById("feedback-a") as HTMLAnchorElement;
+    if (feedbackAnchor) {
         feedbackAnchor.href = "./feedback.html?uuid=" +
-            encodeURIComponent(request_uuid) + "&hash=" +
-            encodeURIComponent(hash(request)) + "&serverURL=" +
-            encodeURIComponent(serverUrl);
-        feedbackAnchor.target = "_blank";
-        footer.appendChild(feedbackAnchor);
+                encodeURIComponent(request_uuid) + "&hash=" +
```

Comment:
not blocking, but may be you can also remove feedbackLinkText from messages.json.